Druid/DRCogitJITConstantSplitting.class.st

```diff
@@ -1,35 +0,0 @@
-"
-Since in cogit we cannot express operations with all constant operands,
-We need to split instructions in two.
-E.g.,
-	
-	A := 1 + 2
-	=>
-	B := 1
-	A := B + 2
-"
-Class {
-	#name : #DRCogitJITConstantSplitting,
-	#superclass : #DROptimisation,
-	#category : #'Druid-Cogit'
-}
```

Comment:
Ahh ok... this is not necessary anymore 👍

Reply:
Yup